worker/src/DepLibUring.cpp

```diff
@@ -313,15 +313,15 @@ DepLibUring::LibUring::LibUring()
 
 	if (err < 0)
 	{
-		int errno = -err;
+		int errno = -1 * err;
```

Comment:
The problem here seems the variable name. It's not a good idea using `errno` as a local variable as the compiler threats it specially (see the compiling errors.)

Reply:
Also, this file is using `-err` in other places for logging purposes, so this change is generating an inconsistency across the file.

Reply:
Changes done to mak it consistent.